server/src/test/java/org/opensearch/search/profile/ProfileResultTests.java

```diff
@@ -116,6 +116,11 @@ private void doFromXContentTestWithRandomFields(boolean addRandomFields) throws
             assertNull(parser.nextToken());
         }
         assertEquals(profileResult.getTime(), parsed.getTime());
+        assertEquals(profileResult.getSliceStartTime(), parsed.getSliceStartTime());
+        assertEquals(profileResult.getMaxSliceTime(), parsed.getMaxSliceTime());
+        assertEquals(profileResult.getMinSliceTime(), parsed.getMinSliceTime());
+        assertEquals(profileResult.getAvgSliceTime(), parsed.getAvgSliceTime());
+        assertEquals(profileResult.isConcurrent(), parsed.isConcurrent());
```

Comment:
Can we add test for case when concurrent is `false` and the output doesn't contain the `min/max/avg` slice times.

Reply:
We need to also add tests for `to and fromXContext` for concurrent search case as well with new fields.